### ISSUE TITLE
Conflict with markdown-it-katex

### ISSUE THREAD

**Issue by phoihos** [arve0/markdown-it-attrs#115]
There is a syntax conflict with [waylonflinn/markdown-it-katex](https://github.com/waylonflinn/markdown-it-katex) or other forked version [@neilsustc/markdown-it-katex](https://www.npmjs.com/package/@neilsustc/markdown-it-katex):

```js
var md = require('markdown-it')();

md.use(require('markdown-it-katex'));
md.use(require('markdown-it-attrs'));

var result = md.render('$\\sqrt{a}$');
console.log(result) // outputs <p a="">\sqrt</p>
```

Workaround: [Using custom delimiters](https://github.com/arve0/markdown-it-attrs#custom-delimiters)

Is there a way to resolve this without changing the delimiters?

**Comment by arve0** [comment]
Hi 👋 The pattern that markdown-it-attrs are looking for, is `{a}` in the end of an element. This does not make a conflict:

```js
md.render('$\\sqrt{a}$ asdf');
```

https://runkit.com/arve0/5ff55bb7101b92001291ee45

Maybe it's possible to change the order of attrs or katex to resolve it. Order can be adjusted here: https://github.com/arve0/markdown-it-attrs/blob/master/index.js#L40

**Comment by phoihos** [comment]
OK. I've got this.

> Maybe it's possible to change the order of attrs or katex to resolve it.

**Comment by sajozsattila** [comment]
I could not manage to solve the problem with the order change.  Is anybody have success with it? 

What could be the optimal solution for the issue? I guess somehow ignore the match between $$ and $ characters.

**Comment by ickc** [comment]
Confirm that this conflicts with markdown-it-texmath too.

**Comment by arve0** [comment]
This should have been resolved in [v4.1.0](https://github.com/arve0/markdown-it-attrs/releases/tag/v4.1.0). Please reopen if not.

**Comment by mb21** [comment]
Thanks a lot! seems to work for me.
